web/beacon-app/src/features/teams/components/NewUserInvitationPage.tsx

```diff
@@ -16,12 +16,14 @@ export function NewUserInvitationPage({ data }: { data: any }) {
 
   const invitee_token = searchParams.get('token');
   const handleSubmitRegistration = (
-    values: NewInvitedUserAccount,
-    helpers: FormikHelpers<NewInvitedUserAccount>
+    values: NewUserAccount,
+    helpers: FormikHelpers<NewUserAccount>
   ) => {
     const payload = {
       ...values,
-      organization: data.org_name,
+      privacy_agreement: true,
+      terms_agreement: true,
+      organization: data.org_name, // WHY THIS HAS BEEN ADDED?
```

Comment:
This was included in the following PR: https://github.com/rotationalio/ensign/pull/392